sdk/src/main/java/io/opentelemetry/sdk/metrics/MeterSdkProvider.java

```diff
@@ -42,11 +44,12 @@
 
   private final MeterSdkComponentRegistry registry;
   private final MetricProducer metricProducer;
+  private final ViewRegistry viewRegistry = new ViewRegistry();
 
   private MeterSdkProvider(Clock clock, Resource resource) {
     this.registry =
         new MeterSdkComponentRegistry(
-            MeterProviderSharedState.create(clock, resource), new ViewRegistry());
+            MeterProviderSharedState.create(clock, resource), viewRegistry);
```

Comment:
Wondering why we don't want ```ViewRegistry``` per ```MeterSdk``` instance level? What if I have Metric instruments with the same type and name but I want it have different custom aggregation & temporality on different ```instrumentationName```?

Reply:
The SDK user won't in general know what named Meters are available, so I don't know how you would access the specific Meter in order to have a registry at that level. How were you thinking that an operator would know how to use this if the API were available only on the individual Meters?